### DISCUSSION TITLE
can mdx be used with markdown which we pass from our server over the graphql api?

### DISCUSSION THREAD

**Discussion by jlorezz** [mdx-js/mdx#1843] [Q&A] [answered]
can mdx be used with markdown which we pass from our server over the Graphql API or will it only work with files that are in the frontend?

**Comment by ChristianMurphy** [comment, marked as answer]
MDX can work client side or server side.
It can use markdown from files or from a plain string (which could come from anywhere including graphql).
There are guides introducing the various integrations on the mdx website https://mdxjs.com/packages/

**Reply by jlorezz** [reply]
sick! thanks a lot!
